app.properties

```diff
@@ -0,0 +1,101 @@
+# =========================================================================
+# this is a simple properties file
+# meant to curate important meta data
+# about what a team is building
+#
+# each entry should have a single value
+# or a list of comma separated values
+#
+# this file can be filled incrementally
+# since fields like app.urls (links to the deployed solutions)
+# might not have been created right now, so you can leave ot blank.
+#
+# the file should be at the root of a repo
+# =========================================================================
+
+
+# about the product
+# ================= 
+
+# name of the product 
+name: SlumData
+
+# what SDG goal(s) does the product solve
+sdgs: 17
+
+# name (not link) of repo
+repos: Team-269-Backend
+
+# URL(s) to the deplyed
+# app for the solution
+# e.g frontend UI app pr backend API
+app.urls:
+
+# URL(s) to the Codacy 
+# dashbaord for this repo
+# e.g https://app.codacy.com/gh/BuildForSDG/farmify-frontend/dashboard
+
+codacy.urls: https://app.codacy.com/gh/BuildForSDG/Team-269-Backend/dashboard?bid=18198678
+
+# YES if the solution
+# is solving for COVID-19
+# otherwise, NO
+app.forCOVID19: NO
+
+# what industries
+# does this solution
+# or its problem statement fall
+# under, e.g Agriculture
+# pick from this list : https://www.opensecrets.org/industries/alphalist.php
+app.industries: Government
+
+# about the codebase
+# ================== 
+
+# what are the major 
+# coding languages used
+# to create the solution
+code.languages: PHP
+
+# what are the major 
+# frameworks and libraries
+# used to create the solution
+code.frameworks: Laravel
+
+# what Facebook open source framework
+# library, tool, or platform API
+# is the codebase using
+code.opensource.facebook: React.js
+
+# If the team created something
+# as a by-product of building the 
+# product, and that something is now
+# open shource, pls list them 
+code.opensource.contributions:
+
+# about the team
+# ==============
+
+# name of the team
+team.name: Team-229
+
+# github username
+# of the team's TTL(s)
+team.ttls: onabright
+
+# github username
+# of the team's mentor(s)
+team.mentors:
```

Comment:
I think we are missing the team mentor details...

Reply:
@geneowak yes details are missing. I have just got them! Let me update